README.adoc

```diff
@@ -162,6 +162,13 @@ Raspberry Pi devices that use an EEPROM as part of their boot flow can configure
 
 Set to any value to enable EEPROM write protection.
 
+=== RPI_SB_PROVISONER_MANUFACTURING_DB
+*Optional*
+
+Store manufacturing data in a sqlite3 database. This will include the board serial, board revision, the boot ROM version, the MAC address of the ethernet port, any set hash of the customer signing key, the JTAG lock state, the board attributes and the advanced boot flags.
+
+Set to a writeable file path to enable data storage.
```

Comment:
Should document that this must not be on a network filesystem as WAL is being used.